tagging.md

```diff
@@ -533,8 +533,9 @@ This corresponds to this layout of EBML elements:
 
 # Official Tags
 
-The following is a complete list of the supported Matroska Tags. While it is possible
-to use Tag names that are not listed below, this is **NOT RECOMMENDED** as compatibility will
+The following is a complete list of the supported Matroska Tags.
+As stated in (#why-official-tags-matter) it is better to use official tags in this as much as possible,
```

Comment:
```suggestion
As stated in (#why-official-tags-matter) it is better to use official tags as much as possible,
```

Reply:
Applied. I don't remember what I want to express with "this".